egs/dihard_2018/v1/local/nnet3/xvector/prepare_feats.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
```

Comment:
You don't need v1/local/nnet3, since you already have this directory in v2/local/nnet3. You should delete it.